### PR TITLE
Water facility restoration model #55

### PR DESCRIPTION
**Pull request by longshuicy**

### CODE REVIEW

File: manual_jb/content/analyses/water_facility_restoration.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+--- | --- | --- | ---
+`dfr3_mapping_set` <sup>*</sup> | `incore:dfr3MappingSet` | DFR3 Mapping Set | DFR3 Mapping Set.
+
+**Output datasets** 
+
+key name | type | parent key | name | description
+--- | --- | --- | --- | ---
+`time_results` <sup>*</sup> | `incore:waterFacilityRestorationTime` | `water_facilities` | Time results | A csv file recording repair time at certain functionality recovery for each class and limit state.
+`pf_results` <sup>*</sup> | `incore:waterFacilityRestorationFunc` | `water_facilities` | Percentage of functionality results | A csv file recording functionality change with time for each class and limit state
+<small>(* required)</small>
```
Comment (ncsa-mo) [inline manual_jb/content/analyses/water_facility_restoration.md:35]: Please, add empty line above the star.

File: manual_jb/content/analyses/water_facility_restoration.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Water facility restoration
+
+**Description**
+
+This analysis computes the repair time, and the percentage of functionality change with time for water facility 
+restoration based on a particular hazard. Currently supported hazard is: **earthquake**.
```
Comment (vnarah2) [inline manual_jb/content/analyses/water_facility_restoration.md:6]: I think the supported hazards should the ones supported by Water facility damage. I don't see anything specific to EQ in this analysis except we are using an eq specific mapping from the database. Tagging @navarroc for his input

**Review by vnarah2 — CHANGES_REQUESTED** [review]

**Review by ncsa-mo — APPROVED** [review]

File: manual_jb/content/analyses/water_facility_restoration.md (hunk 2 above)
Comment (longshuicy) [reply manual_jb/content/analyses/water_facility_restoration.md:6]: oh i put down earthquake cuz that's the curves I have right now. Not sure what it should be

Comment (navarroc) [reply manual_jb/content/analyses/water_facility_restoration.md:6]: Are the restoration curves specific to earthquake? The curves look to be in the form of X amount of damage it takes Y time to repair so it seems generic and could take any water facility damage we could support.

Comment (vnarah2) [reply manual_jb/content/analyses/water_facility_restoration.md:6]: I put this in pipeline analysis. if that's acceptable we can use similar format for other restoration analyses
```
This analysis computes water pipeline restoration time after a particular hazard. The analysis itself is hazard agnostic, as it works
 with the output of [Pipeline damage with repair rate analysis](pipeline_dmg_w_repair_rate.md) and restoration curves.
```

File: manual_jb/content/analyses.md [hunk 3]
```diff
@@ -25,3 +25,4 @@
 23. [Tornado electric power network (EPN) damage](analyses/tornadoepn_dmg)
 24. [Transportation recovery](analyses/transportation_recovery)
 25. [Water facility damage](analyses/waterfacility_dmg)
+25. [Water facility restoration](analyses/water_facility_restoration)
```
Comment (navarroc) [inline manual_jb/content/analyses.md:28]: I think this needs to be 26?

**Comment by navarroc** [comment]
You need to add a reference to your md file in _toc.yaml file so the analysis shows up in the sidebar of the documentation under pyincore > Analyses.

**Review by navarroc — APPROVED** [review]

**Review by vnarah2 — CHANGES_REQUESTED** [review]

**Comment by longshuicy** [comment]
> You need to add a reference to your md file in _toc.yaml file so the analysis shows up in the sidebar of the documentation under pyincore > Analyses.

done~

**Pull request merged**